src/main.js

```diff
@@ -59,15 +74,35 @@ const parseVarFromType = envSpecAsArray => {
 const outputHTML = envValues => {
   //create HTML format
   if (envValues) {
-    envValues = envValues.map(
-      element =>
-        `<label for="env_spec_${element[0].toLowerCase()}">${
-          element[0]
-        }</label>\n` +
-        `<input id="env_spec_${element[0].toLowerCase()}" name="${element[0].toLowerCase()}" type="${
-          element[1]
-        }" />\n`
-    );
+    envValues = envValues.map(element => {
+      switch (element[2]) {
+        case "hasType": //if element is value with type
+          toPrint =
+            `<label for="env_spec_${element[0].toLowerCase()}">${
```

Comment:
Since you are printing the same label for both typed entries and restricted choices, we should do one of the following:

1. Extracting this logic to a separate function (e.g. `renderLabelForEntry(entry)`)
2. Running this kind of logic outside the `if/switch` clause

Reply:
I have a question, in your examples there is a different output for syntax with types and for syntax with choices.
In issue#3 https://github.com/sourcelair/env-spec-js/issues/3
input: ADMIN_EMAIL: email
output:
<label for="env_spec_admin_email">ADMIN_EMAIL</label>
<input id="env_spec_admin_email" name="admin_email" type="email" />

and in issue#4 https://github.com/sourcelair/env-spec-js/issues/4
input: DEBUG: [0,1]
output:
<label for="env_spec_admin_debug">DEBUG</label>
<select id="env_spec_admin_debug" name="admin_debug">
  <option value="0">0</option>
  <option value="1">1</option>
</select>

In the second example there is an extra admin word  in lines :<label for="env_spec_admin_debug">DEBUG</label> and <select id="env_spec_admin_debug" name="admin_debug">
did this happen by mistake?